packages/iTwinUI-react/src/core/ComboBox/ComboBox.tsx

```diff
@@ -30,6 +32,54 @@ import { ComboBoxInput } from './ComboBoxInput';
 import { ComboBoxInputContainer } from './ComboBoxInputContainer';
 import { ComboBoxMenu } from './ComboBoxMenu';
 import { ComboBoxMenuItem } from './ComboBoxMenuItem';
+import SelectTagContainer from '../Select/SelectTagContainer';
+import SelectTag from '../Select/SelectTag';
+
+const isMultipleEnabled = <T,>(
+  variable: (T | undefined) | (T[] | undefined),
```

Comment:
what is "variable"?

Reply:
It is value we are testing. Eg. valueProp or selectedIndex can be both T and T[]. So this way we can check types of these variables. (copied from multiselect)